### ISSUE TITLE
Semester Research 

### ISSUE THREAD

**Issue by irajmoradi** [mikeizbicki/modulus-magnus-linguae#73]
Hey Mike,

Aly and I talked today and were wondering if you have any time next week to meet with us. Furthermore we feel like it would be beneficial to set up a weekly meeting time to talk about research. 

On another note both Aly and I are interested in getting credit for doing research this semester. I need a syllabus and Aly needs learning outcomes and method of evaluation on her form. I can create a draft syllabus if that would help. Aly emailed her form, mine is attached here. 

[IRAJRESEARCH.pdf](https://github.com/mikeizbicki/modulus-magnus-linguae/files/12481486/IRAJRESEARCH.pdf)

Sincerely,
Iraj James Moradi

**Comment by mikeizbicki** [comment]
Yes, I'd be happy to supervise a research course with you both and yes, we should setup a time to meet.  I'm trying to arrange all of my meetings to be on Wednesdays.  Would 9am-10am work for you both?  If not, let me know your availability.

Also, I will look into your forms here sometime soon (and feel free to pester me about it if you're running out of time).  I've got a huge backlog of email that I'm slowly working my way through.

**Comment by lwelch25** [comment]
Hey Mike, can I also get research course credit while abroad? I'll be continuing research with aly and Iraj while in Budapest for the fall. I'm more than happy to meet over zoom during that time as well.

**Comment by mikeizbicki** [comment]
I'd be fine with that, but the registrar might not be.  I'd just try to submit the form without mentioning anything about being abroad and seeing what happens.

We'll do the whole group meeting over zoom.

**Comment by alysawyer** [comment]
That sounds great, thank you! Looking forward to it.

Unfortunately Wednesdays are my busiest days class wise-- the longest break I have from 9am-4pm is 30 minutes (for lunch). So I can do before 9 or after 4 (though I can't do from 5-6).

Regarding the form, if you could send it back to me in the next week that would be best as the Harvey Mudd add deadline is approaching.

**Comment by irajmoradi** [comment]
Specifically for the reading list and syllabus required for the forms would you like us to begin developing it or would that not be helpful. Also for me I need it during next week as well.

**Comment by lwelch25** [comment]
I also need a syllabus and reading list to submit my request to the registrar. More than happy to create the syllabus. However, I'm not sure if I'd be the best person to create the reading list. Thank you Mike!

**Comment by mikeizbicki** [comment]
Eeek... I'm sorry I somehow missed these messages until now.

Is there a different day that 9am meetings would work for everyone?  Say Thursday?

In any event, we won't meet tomorrow morning, but I'll use that time to get everyone's paperwork complete and send it to you.

**Comment by alysawyer** [comment]
9am on Thursdays work for me, typically (can't do this week as I have a meeting already scheduled then)

Getting the paperwork done by tomorrow sounds great! Thank you!

**Comment by irajmoradi** [comment]
I can’t do 9 am any day besides Friday and weekends (I either have class at
9:35 mw or class that ends at 9:25 Tuesday Thursday).

**Comment by mikeizbicki** [comment]
It looks like scheduling will be complicated enough that we should use when2meet.  You can find it at: <https://www.when2meet.com/?21223143-GgLez>.  This week I'd be free to meet tomorrow, but not on Friday.  It's okay if we have to start our first meeting next week.

Iraj/Lucas, the following language should work for your submission forms:

Syllabus:

We will meet weekly for 1hr/week, and the student will be expected to work for an additional 9hr/week.  The additional work will include programming projects, reading, and technical writing.  The final goal of the course will be to prepare a research paper for submission to a computer science venue.  The students will be evaluated on their ability to make progress towards this goal.

Readings:

We will read relevant research papers from ICML, NeurIPS,
ICLR, and other machine learning and natural language
processing venues.

**Comment by irajmoradi** [comment]
[research combined.pdf](https://github.com/mikeizbicki/modulus-magnus-linguae/files/12541345/research.combined.pdf)

Can you sign the final pdf if it looks alright? I have it at a full course credit if that works as well, and added a page with the language you used.

**Comment by irajmoradi** [comment]
> Also, I will look into your forms here sometime soon (and feel free to pester me about it if you're running out of time). I've got a huge backlog of email that I'm slowly working my way through.

The add deadline is monday and I have to still get my advisor to sign it and so I would want it by today or tommorow to ensure my advisor has time to sign it by monday. You can also email it to me instead of sending it through github issues if that works better.

**Comment by irajmoradi** [comment]
I got this response from the registrar in regard to the independent study form. 

<img width="1117" alt="Screenshot 2023-09-08 at 10 54 45 AM" src="https://github.com/mikeizbicki/modulus-magnus-linguae/assets/123036441/359118a2-f825-4897-91c7-27cde5e7ecc4">

**Comment by mikeizbicki** [comment]
@irajmoradi It should be fine, but there's also now  MATH197A CM-01 listed in the portal as a research class with me that you can probably just sign up for directly if not.

@lwelch25 I don't see that you've submitted your availability on when to meet.  I 100% understand if you're not able to find time since you're abroad and should be focusing on getting the most out of that experience right now.

In any event, why don't we do 9:40AM on Tuesday to meet?  That should give you all enough time between classes.

**Comment by lwelch25** [comment]
My apologies. I just submitted my available times on when2meet.

**Comment by mikeizbicki** [comment]
Okay, let's change the meeting time to Thursday@12:15.  We'll meet over zoom at https://cmc-its.zoom.us/j/3952597181

**Comment by irajmoradi** [comment]
Hey Mike, I just got this email for curriculum committee for the Independent Study:

"
I’m a member of the Curriculum Committee. I need the following information in order to be able to recommend approval:


  1.  How the I.S. will be evaluated (each assignment or aspects of the paper, project, video, or artwork is worth a total of points or a percentage as well as total points needed for the course)


Just a basic breakdown of percentages for the assignments will be fine. Please respond ASAP so we can quickly process the request.
"


Would this breakdown for percentages work for them?


Paper Submission for conference (40%)
Weekly Progress Updates on Research including results from code (20%)
Active Participation on research forum (Github Issues) with other students (20%)
Reading discussions and reflections (20%)

**Comment by irajmoradi** [comment]
I Need to send it asap so I will send the committee this breakdown at 10:00 AM tommorow

**Comment by mikeizbicki** [comment]
Yes, this is a fine breakdown.
